swig/matlab/pireceiver.m

```diff
@@ -1,4 +1,4 @@
-loadlibrary(GetFullPath('~/local/helics-1.0.0a/lib/libhelicsSharedLib.dylib'));
+loadlibrary(GetFullPath('~/local/helics-develop/lib/libhelicsSharedLib.dylib'));
```

Comment:
Just noticing that these files are specific to your local system and are specific to a mac.  Is there anything we can do to make them more generic?  and why are we loading this library vs something specific to the matlab mex file as this looks like it loads the c shared library?  I would have thought the mex wrapper would take care of that?

Reply:
I was going to update the documentation. The only way I was able to get the MATLAB extension working was to load the shared library manually before calling any HELICS functions. This is why having a `helics-config` binary would be extremely useful.